S/Singular/build_tarballs.jl

```diff
@@ -28,16 +28,16 @@ import Pkg.Types: VersionSpec
 #
 name = "Singular"
 
-upstream_version = v"4.4.0-7" # 4.4.0p7
-version_offset = v"0.0.10"
+upstream_version = v"4.4.0-6" # 4.4.0p6
+version_offset = v"0.0.11"
```

Comment:
I only just saw that the VersionNumber scheme here includes the pre-release data, so we would need one of the following two to get a higher version than `404.0.710`
```suggestion
upstream_version = v"4.4.0-6" # 4.4.0p6
version_offset = v"0.0.111"
```
or
```suggestion
upstream_version = v"4.4.0-7" # 4.4.0p6 (the difference here is due to a retroactive re-release of 404.0.606 as 404.0.711)
version_offset = v"0.0.11"
```